<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>FTP server:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ftpserverlineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>ftp.qt.nokia.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>user:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNamelineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>the default username is anonmous, the psw is any</string>
   </property>
   <property name="text">
    <string>anonymous</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pswlineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>123456</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>psw:</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="fileList">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>256</width>
     <height>161</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>file</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>size</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>owner</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>group</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>modify</string>
    </property>
   </column>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="cdToParentButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>cd to parent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>download</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
